app/components/tables/holidays_table/table_element/FilterBar.tsx

```diff
@@ -0,0 +1,22 @@
+import React, {useState} from "react";
+
+interface filterBarProps {
+    filtering : any
+    setFiltering : any
```

Comment:
Let's avoid using the `any` type. The filtering type should be `string` and I believe the type of `setFiltering` should be `Dispatch<SetStateAction<string>>`